paimon-core/src/main/java/org/apache/paimon/mergetree/compact/PartialUpdateMergeFunction.java

```diff
@@ -379,6 +380,16 @@ private Map<Integer, FieldAggregator> createFieldAggregators(
                                     isPrimaryKey,
                                     options,
                                     fieldName));
+                } else if (defaultAggFunc != null) {
```

Comment:
Can you also modify `AggregateMergeFunction`?

Reply:
@JingsongLi `AggregatedMergeFunction` is updated too